web/vtadmin/src/components/routes/tablet/FullStatus.tsx

```diff
@@ -30,8 +44,22 @@ const FullStatus: React.FC<Props> = ({ tablet }) => {
             </div>
         );
     }
+    if (!isNil(data) && !isNil(data.status)) {
+        if (data.status.semi_sync_primary_enabled !== true) {
+            data.status.semi_sync_primary_enabled = false;
+        }
```

Comment:
Here if you are trying to set `data.status.semi_sync_primary_enabled` to a boolean, you can simplify by doing:
```suggestion
        data.status.semi_sync_primary_enabled = !!data.status.semi_sync_primary_enabled
```

Reply:
Fixed